package.json

```diff
@@ -3,13 +3,15 @@
   "version": "2.10.1",
   "description": "A distributed networking stack for connecting peers",
   "main": "swarm.js",
+  "types": "index.d.ts",
   "dependencies": {
     "@hyperswarm/network": "^1.2.0",
     "shuffled-priority-queue": "^2.1.0",
     "utp-native": "^2.1.3"
   },
   "devDependencies": {
     "@hyperswarm/dht": "0.0.1",
+    "@types/node": "^13.11.0",
```

Comment:
Why is this needed and how do you make sure it's up to date with all active Node versions?